CONTRIBUTING.md

```diff
@@ -86,12 +86,47 @@ A lot of the instructions below replicate what the [PR CI pipeline] is doing.
 1. You are now ready to submit your PR.
 1. After your PR ise merged, most likely you will want to read [`How to deploy your changes`](#how-to-deploy-your-changes) to verify they got deployed.
 
-# New linter rule promotion strategy
+# (WIP) New linter rule promotion strategy
 
-1. Merge new rules to main. As a result, they show up in the Staging LintDiff pipeline.
+1. __MAYBE__: run the new rule (only the new rule) locally for all of the existing swagger specs and aggregate the results
+1. Merge new rules to main
+    1. As a result, they show up in the Staging LintDiff pipeline.
+    1. __MAYBE__: pipeline automatically runs the linter on all the existing API specs for new linter rule merges to main.
 1. Review Staging LintDiff pipeline build logs to see if the rules work correctly.
-1. Once you verify the rules work correctly, roll them out to the production pipeline following the process defined in [How to deploy your changes](#how-to-deploy-your-changes).
-1. If after promoting the rule you find it is not behaving correctly, disable it via the process defined in [How to disable or enable existing Spectral rules](#how-to-disable-or-enable-existing-spectral-rules).
+    1. This can be done via Kusto (__TODO__: write a query).
+    1. __TODO__: decide how to determine a rule is working correctly
```

Comment:
We might be missing some logs that explicitly state "this rule ran and succeeded". If so, I can assist in getting them in.